projects/packages/image-cdn/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.2.4] - 2023-08-28
 ### Changed
 - Updated package dependencies. [#32605]
+- Rely on WordPress HTML API to parse HTML instead of Regex Patterns. [#43743]
```

Comment:
is there a way to do this without running `jetpack` on my machine? I don't have `wp-cli` or Docker or know what the `jetpack` command is.

Reply:
There is an introduction to the Jetpack development environment here https://github.com/Automattic/jetpack/blob/trunk/docs/development-environment.md#clone-the-repository and also a [Quick Start Guide here](https://github.com/Automattic/jetpack/blob/trunk/docs/quick-start.md)

Reply:
> is there a way to do this without running jetpack on my machine?

If you'd rather not install [the Jetpack CLI tool](https://github.com/Automattic/jetpack/blob/trunk/tools/cli/README.md) (although it would be useful for other things like running the tests with `jetpack test packages/image-cdn php`), and have `composer` installed on your machine, you should be able to run the changelogger commands directly. The Changelog tool is [a composer package](https://packagist.org/packages/automattic/jetpack-changelogger) that can be installed in any project, and it is a dependency in the image-cdn package where you're working:
https://github.com/Automattic/jetpack/blob/51bb232b314a6cee550d2072d5bfe0b341d97c27/projects/packages/image-cdn/composer.json#L13

In practice, you'd run `composer install` in `projects/packages/image-cdn`, and then run `vendor/bin/changelogger add` to create a new changelog entry.

Reply:
thank you @jeherve !!!

Reply:
Happy to help! And now you see the consequences of adding a changelog entry specifying a `patch` level change. GitHub checks expect you to bump the package's version to match that patch version increase, and give you another script you can run to do that automatically: `tools/fixup-project-versions.sh`

https://github.com/Automattic/jetpack/pull/32700/files